src/Components/GrowthChartSelector/ChartSelector.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { Menu } from '@headlessui/react';
+import { CategoryCodes } from '../../types/chartDataTypes';
+import { chartData } from '../../DataSets/WhoStandardDataSets/ZScores/ChartDataZscores';
+
+interface ChartSelectorProps {
+    category: keyof typeof CategoryCodes;
+    dataset: keyof typeof chartData;
+    setCategory: (category: keyof typeof CategoryCodes) => void;
+    setDataset: (dataset: keyof typeof chartData) => void;
+}
+
+export const ChartSelector = ({
+    category,
+    dataset,
+    setCategory,
+    setDataset,
+}: ChartSelectorProps) => {
+    const handleCategoryChange = (key: string) => {
+        const newCategory = CategoryCodes[key as keyof typeof CategoryCodes];
+        setCategory(newCategory);
+        setDataset(Object.keys(chartData[newCategory].datasets)[0]);
+    };
+
+    const handleDatasetChange = (key: string) => {
+        const newDataset = key as keyof typeof chartData;
+        setDataset(newDataset);
+    };
+
+    return (
+        <div className='w-full flex'>
+            <div className='flex gap-2 text-sm pl-12'>
+                <div className='flex flex-col relative min-w-fit'>
+                    <Menu>
+                        {({ open }) => (
+                            <>
+                                <Menu.Button
+                                    className={'flex flex-row rounded w-full border border-gray-300 py-1 gap-2 px-4'}
+                                >
+                                    {category}
+                                    <svg
+                                        xmlns='http://www.w3.org/2000/svg'
+                                        fill='currentColor'
+                                        id='Outline'
+                                        viewBox='0 0 24 24'
+                                        className={`w-5 h-5 ${open ? 'rotate-180' : ''}`}
```

Comment:
Here

Reply:
Consider making these match
![image](https://github.com/dev-otta/dhis2-who-growth-chart/assets/136808617/39f7586e-dbeb-44ab-892b-2695ab63d2d8)
Because the buttons look the same